app/src/main/java/com/p2p/presentation/tuttifrutti/finalscore/FinalScoreTuttiFruttiFragment.kt

```diff
@@ -0,0 +1,59 @@
+package com.p2p.presentation.tuttifrutti.finalscore
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.p2p.R
+import com.p2p.databinding.FragmentTuttiFruttiFinalScoreBinding
+import com.p2p.presentation.basegame.BaseGameFragment
+import com.p2p.presentation.extensions.requireValue
+import com.p2p.presentation.tuttifrutti.TuttiFruttiViewModel
+
+class FinalScoreTuttiFruttiFragment : BaseGameFragment<
+        FragmentTuttiFruttiFinalScoreBinding,
+        TuttiFruttiFinalScoreEvent,
+        FinalScoreTuttiFruttiViewModel,
+        TuttiFruttiViewModel>() {
+    override val gameViewModel: TuttiFruttiViewModel by activityViewModels()
+
+    override val gameInflater: (LayoutInflater, ViewGroup?, Boolean) -> FragmentTuttiFruttiFinalScoreBinding =
+        FragmentTuttiFruttiFinalScoreBinding::inflate
+
+    override val viewModel: FinalScoreTuttiFruttiViewModel by viewModels()
+
+    private lateinit var tuttiFruttiFinalScoreAdapter: TuttiFruttiFinalScoreAdapter
+
+    override fun initUI() {
+        super.initUI()
+        setupObservers()
+        setupScoreRecycler()
+        gameBinding.exitButton.setOnClickListener { viewModel.exit() }
+    }
+
+    override fun setupObservers() = with(viewModel) {
+        super.setupObservers()
+        finalScores.observe(viewLifecycleOwner) {
```

Comment:
```suggestion
        finishedRoundsPointsInfos.observe(viewLifecycleOwner) {
```
Fijate que en mi pr que ya estableci la estructura de esto! Asi despues no se hace doloroso el merge